### PR TITLE
Boundary integral file time step 309

### PR DESCRIPTION
**Pull request by ktbolt**
I've fixed writing integral quantities (see https://github.com/SimVascular/svMultiPhysics/issues/309).

I've also updated `svMultiPhysics/tests/cases/fluid/pipe_RCR_3d` to write integral quantities since this is not tested anywhere.

### CODE REVIEW

**Comment by ktbolt** [comment]
The errors are caused by exceptions thrown by the integration functions (e.g. `all_fun::integ()`).  This was caused by initializing the output types to true like the original code did, changed that.

File: Code/Source/solver/read_files.cpp [hunk 1]
```diff
@@ -2243,35 +2243,37 @@ void read_outputs(Simulation* simulation, EquationParameters* eq_params, eqType&
   // 
   for (int j = 0; j < 3; j++) {
     for (int k = 0; k < nDOP[j+1]; k++) {
-      lEq.output[k].wtn[j] = true;
+      lEq.output[k].options.boundary_integral = false;
```
Comment (dcodoni) [inline Code/Source/solver/read_files.cpp:2246]: @ktbolt is the loop on j still necessary here?

**Review by dcodoni — COMMENTED** [review]
Very good! I only have one comment/question.

File: Code/Source/solver/read_files.cpp (hunk 1 above)
Comment (ktbolt) [reply Code/Source/solver/read_files.cpp:2246]: @dcodoni It was not clear to me from the code why all of the `wtn[j]` were set to true so I left the loop. But the `boundary_integral` are all initialized to false anyway so I will remove the loop.

**Review by dcodoni — APPROVED** [review]

**Pull request merged**